cpp/ql/test/library-tests/ir/ir/raw_ir.expected

```diff
@@ -5277,8 +5277,8 @@ ir.cpp:
 # 1153|     mu0_2(unknown)                                     = UnmodeledDefinition              : 
 # 1153|     r0_3(glval<int>)                                   = VariableAddress[i]               : 
 # 1153|     mu0_4(int)                                         = InitializeParameter[i]           : &:r0_3
-# 1154|     r0_5(glval<__attribute((vector_size(16))) int>)    = VariableAddress[vi4]             : 
-# 1154|     mu0_6(__attribute((vector_size(16))) int)          = Uninitialized[vi4]               : &:r0_5
+# 1154|     r0_5(glval<__attribute((vector_size(16UL))) int>)  = VariableAddress[vi4]             : 
```

Comment:
These types were all returned from `Type.getUnspecifiedType()`. Does that mean that all unspecified types for vector types will always print the size with the `UL` suffix, regardless of how they were declared?

I'm fine with the change in the string. I just want to make sure there's not some underlying issue with `getUnspecifiedType()` that's causing the change.

Reply:
See the library-tests/vector_sizes qltest I added in this patch for the behaviour.  It depends on how the `vector_size ()`attribute was declared.  Using `sizeof()` (which is common) will make the type `unsigned long` (in my environment)